consumer/consumer_internal_test.go

```diff
@@ -9,9 +13,111 @@ import (
 	cluster "github.com/bsm/sarama-cluster"
 	"github.com/stretchr/testify/require"
 
+	"github.com/heetch/felice/consumer/handler"
 	"github.com/heetch/felice/message"
 )
 
+const (
+	logRegexPrefix = "\\[Felice\\] [0-9]*/[0-1][0-9]/[0-3][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] "
+)
+
+// TestLogger grabs logs in a buffer so we can later make assertions
+// about them.
+type TestLogger struct {
+	buf    *bytes.Buffer
+	Logger *log.Logger
+	t      *testing.T
+}
+
+// NewTestLogger constructs a test logger we can make assertions against
+func NewTestLogger(t *testing.T) *TestLogger {
+	tl := &TestLogger{
+		buf: bytes.NewBuffer([]byte{}),
```

Comment:
:tulip: :book: For info, the zero value of `bytes.Buffer` is a valid buffer

Reply:
:+1:

Reply:
.. ah yea, it was because I was using a pointer which is implied by `log.New(..` but a handly `&` will do just as well :-)